### PR TITLE
too much D:

### PR DESCRIPTION
**Pull request by bukzor-sentryio**
- auto-commit: .github, for test
- auto-commit: .github, for test
- too much D:

### CODE REVIEW

File: manual_tests/behaviors/lock_on_pr.py [hunk 1]
```diff
@@ -5,10 +5,5 @@
 
 
 def test(pr: tacos_demo.PR) -> None:
-    for slice in pr.slices:
-        assert (
-            pr.check("Terraform Lock", f"tacos-gha / main ({slice})")
-            .wait()
-            .success
-        )
+    assert pr.check("tacos_lock").wait().success
```
Comment (IanWoodard) [inline manual_tests/behaviors/lock_on_pr.py:8]: Did something change that I am not seeing? Why are we no longer checking locking per slice?

**Review by IanWoodard — COMMENTED** [review]
This is a lot of refactor right before the demo tomorrow. Could we just make the necessary changes for now and focus on refactoring down the road?

**Review by IanWoodard — APPROVED** [review]

File: manual_tests/behaviors/lock_on_pr.py (hunk 1 above)
Comment (bukzor-sentryio) [reply manual_tests/behaviors/lock_on_pr.py:8]: changes in slice.py get us much closer to the old semantics: `\.check("X")` will match any job where `run.job.startswith("X")`, with `run.job` defined as the first bit after `" / "` in the check name. In total, it matches all of the tacos_lock jobs at once.

`.wait` is modified to return just one of the jobs matched, as long as they've all finished, with priority given to failed runs.

In total, it means that this line asserts that _all_ "tacos_lock" checks have finished and succeeded.

File: manual_tests/behaviors/unlock_on_merge.py [hunk 2]
```diff
@@ -9,30 +9,16 @@
...
-            .wait(since=lock_run.completed, timeout=30)
-            .success
-        )
-
-
 @pytest.mark.xfail(raises=XFailed)
 def test(pr: tacos_demo.PR) -> None:
-    # TODO: use slice name
-    assert_locked(pr)
+    assert pr.check("tacos_lock").wait().success
```
[the remark this answers is not in the document]
Comment (bukzor-sentryio) [reply manual_tests/behaviors/unlock_on_merge.py:14]: Explained below.

**Comment by bukzor-sentryio** [comment]
> This is a lot of refactor right before the demo tomorrow. Could we just make the necessary changes for now and focus on refactoring down the road?

I  agree that this not ideal, and I also don't like it, but facts are that the demo will only be possible if I don't take the time to separate these changes right now. I apologize and will strive to do better.

**Pull request merged**